ieeeuoft/newhacks-site-2022
# NH-27: Hid remaining stock number from participant side

## Overview

- Resolves NH-27
- When participants look at the remaining stock, they do not need to see the maximum number. For example, 10 out of 20 remaining, the '20' is irrelevant to them
- It is still visible on an admin account


## Unit Tests Created

- n/a (some might be adjusted)


## Steps to QA

- Sign in as a participant and confirm the change, then sign in as admin and it should see the total quantity in stock.
